dev/com.ibm.ws.jpa.tests.jpa_32_fat/test-applications/jakartadata/src/io/openliberty/jpa/data/tests/models/Segment.java

```diff
@@ -22,7 +23,7 @@
  * TODO: uncomment @Entity
  * TODO: remove constructor from Point
```

Comment:
```suggestion
```

Reply:
You resolved the first TODO of commentating the `@Entity` annotation, but did not remove the constructor from Point.